File: java/client/test/com/thoughtworks/selenium/corebased/TestCommandError.java
package com.thoughtworks.selenium.corebased;

import com.thoughtworks.selenium.InternalSelenseTestBase;
import org.junit.Test;

public class TestCommandError extends InternalSelenseTestBase {
	@Test
  public void testCommandError() throws Exception {
		selenium.open("../tests/html/test_verifications.html");
		try { selenium.click("notALink"); fail("expected failure"); } catch (Throwable e) {}
		try { selenium.select("noSuchSelect", "somelabel"); fail("expected failure"); } catch (Throwable e) {}
		try { selenium.select("theSelect", "label=noSuchLabel"); fail("expected failure"); } catch (Throwable e) {}
		try { selenium.select("theText", "label=noSuchLabel"); fail("expected failure"); } catch (Throwable e) {}
	}
}

File: dotnet/test/Selenium.WebDriverBackedSelenium.Tests/TestCommandError.cs
using System;
using System.Collections.Generic;
using System.Text;
using NUnit.Framework;
using System.Text.RegularExpressions;

namespace Selenium.Tests
{
    [TestFixture]
    public class TestCommandError : SeleniumTestCaseBase
    {
        [Test]
        public void CommandError()
        {
            selenium.Open("../tests/html/test_verifications.html");
            try
            {
                selenium.Click("notALink"); Assert.Fail("expected failure");
            }
            catch (Exception) { }

            try
            {
                selenium.Select("noSuchSelect", "somelabel"); Assert.Fail("expected failure");
            }
            catch (Exception) { }

            try
            {
                selenium.Select("theSelect", "label=noSuchLabel"); Assert.Fail("expected failure");
            }
            catch (Exception) { }

            try
            {
                selenium.Select("theText", "label=noSuchLabel"); Assert.Fail("expected failure");
            }
            catch (Exception) { }
        }
    }
}
